electrum/lnpeer.py

```diff
@@ -3070,6 +3073,10 @@ def _check_unfulfilled_htlc_set(
                 return OnionFailureCode.MPP_TIMEOUT, None, None
 
         if mpp_set.resolution == RecvMPPResolution.WAITING:
+            # calculate the sum of just in time channel opening fees
+            htlc_channels = [self.lnworker.get_channel_by_short_id(scid) for scid in set(h.scid for h in mpp_set.htlcs)]
+            jit_opening_fees_msat = sum((c.jit_opening_fee or 0) for c in htlc_channels)
```

Comment:
what is the SCID before the channel gets mined? Or is it already mined at this point? I am confused.

EDIT: oh these are the upstream channels which are already open, not the downstream channel which is the not-yet-mined JIT channel, right?